projects/storefrontapp-e2e-cypress/package.json

```diff
@@ -31,7 +31,9 @@
     "cy:run:ci:cdc-b2b": "cypress run --config-file cypress.ci.json --env BASE_SITE=powertools-spa,OCC_PREFIX_USER_ENDPOINT=orgUsers,API_URL=https://api.cg79x9wuu9-eccommerc1-s1-public.model-t.myhybris.cloud/ --record --key $CYPRESS_KEY --tag \"2211,cdc\" --group CDC --spec \"cypress/integration/vendor/cdc/b2b/*.e2e.cy.ts\"",
     "cy:run:ci:digital-payments": "cypress run --config-file cypress.config.ci.ts --env API_URL=https://backoffice.cp96avkh5f-sapcxteam1-d5-public.model-t.cc.commerce.ondemand.com --record --key $CYPRESS_KEY --tag \"2105,digital-payments\" --group DIGITAL-PAYMENTS --spec \"cypress/e2e/vendor/digital-payments/*.e2e.cy.ts\"",
     "cy:run:ci:epd-visualization": "cypress run --config-file cypress.config.ci.ts --env API_URL=https://api.cp96avkh5f-integrati1-d1-public.model-t.cc.commerce.ondemand.com,BASE_SITE=powertools-epdvisualization-spa --record --key $CYPRESS_KEY --tag \"2105,epd-visualization\" --group EPD_VISUALIZATION --ci-build-id $BUILD_NUMBER --spec \"cypress/e2e/vendor/epd-visualization/**/*.e2e.cy.ts\"",
-    "cy:run:ci:segment-refs": "cypress run --config-file cypress.config.ci.ts --env API_URL=https://api.cg79x9wuu9-eccommerc1-p7-public.model-t.myhybris.cloud/ --record --key $CYPRESS_KEY --tag \"segment-refs\" --group SEGMENT-REFS --spec \"cypress/e2e/vendor/segment-refs/*.e2e.cy.ts\""
+    "cy:run:ci:segment-refs": "cypress run --config-file cypress.config.ci.ts --env API_URL=https://api.cg79x9wuu9-eccommerc1-p7-public.model-t.myhybris.cloud/ --record --key $CYPRESS_KEY --tag \"segment-refs\" --group SEGMENT-REFS --spec \"cypress/e2e/vendor/segment-refs/*.e2e.cy.ts\"",
+    "cy:run:ci:my-account-v2": "cypress run --config-file cypress.config.ci.ts --env API_URL=https://api.cg79x9wuu9-eccommerc1-s5-public.model-t.myhybris.cloud/ --record --key $CYPRESS_KEY --tag \"my-account-v2\" --group MY-ACCOUNT-V2 --spec \"cypress/e2e/regression/my-account/my-account-v2-*.e2e.cy.ts\""
```

Comment:
[MINOR]
Instead of including/excluding tests by the file name, I suggest creating a new folder `/my-account-v2` and cover all tests that live in this folder.
Why I ask: in the next iterations I can imagine more e2e test suites will be added there.

Note: please adjust not only inclusion rule here, but also exclusion rule in the line 20